### ISSUE TITLE
Cannot connect hardware wallet: "Hardware Wallet Connected"

### ISSUE THREAD

**Issue by wjmelements** [MetaMask/metamask-extension#6610]
When I go to "Connect Hardware Wallet" while the Ledger is connected and then click the blue Connect button, it spins for a bit and then a blue notification comes down from the top saying "Hardware Wallet Connected", but I remain on the same page, and there is no way to get to the next page.

Mac + Google Chrome
MetaMask Version: 6.5.1

**Comment by wjmelements** [comment]
![Screen Shot 2019-05-14 at 6 22 52 PM](https://user-images.githubusercontent.com/799573/57742317-866c3380-7675-11e9-8cf2-fac4f3286a4c.png)

**Comment by wjmelements** [comment]
@brunobar79

**Comment by roadh0use** [comment]
same issue here. hardware wallet works on idex and my crypto. metamask when connecting just gives a loading screen followed by "hardware wallet connected". however, nothing connects and the hardware wallet doesnt show up in metamask

**Comment by ethgasstation** [comment]
Same issue here

**Comment by degenspartan** [comment]
same
win 10
chrome Version 74.0.3729.15 / brave Version 0.64.75
MetaMask Version: 6.5.1

**Comment by bandrebandrebandre** [comment]
Same -- macos 10.14.4, chrome 74.0.3729.131, metamask 6.5.1

**Comment by nielsiano** [comment]
Same, MacOS. Tried in latest Chrome and Brave. Getting 2 console errors:

```
Failed to load resource: the server responded with a status of 429 (TOO MANY REQUESTS)
```
and
```
Uncaught TypeError: Cannot read property 'getHostNode' of null
```

**Comment by brunobar79** [comment]
Should be fixed via #6613

**Comment by nielsiano** [comment]
> Should be fixed via #6613

Thanks for the quick fix. Any clues when this will me merged and available?

**Comment by brunobar79** [comment]
@nielsiano  Hopefully today.  In the meantime you can grab a build from here: https://github.com/MetaMask/metamask-extension/pull/6613#issuecomment-492539055

**Comment by TrustHenry** [comment]
same issue here. hardware wallet for Ledger nano s

**Comment by KyleJKistner** [comment]
Same

**Comment by tmashuang** [comment]
Fixed in v6.5.2
